python/shotgun_model/data_handler_cache.py

```diff
@@ -0,0 +1,198 @@
+# Copyright (c) 2016 Shotgun Software Inc.
```

Comment:
yeah, totally agree! Just so hard to know when to stop refactoring... :) This makes it a lot clearer which is great